skaffold-operator.yaml

```diff
@@ -1,253 +1,245 @@
-        - pkg/apis/resources/v1alpha1/helper
-        - pkg/apis/seedmanagement
-        - pkg/apis/seedmanagement/encoding
-        - pkg/apis/seedmanagement/install
-        - pkg/apis/seedmanagement/v1alpha1
-        - pkg/apis/settings
-        - pkg/apis/settings/install
-        - pkg/apis/settings/v1alpha1
-        - pkg/chartrenderer
-        - pkg/client/kubernetes
-        - pkg/client/kubernetes/cache
-        - pkg/controllerutils
-        - pkg/controllerutils/mapper
-        - pkg/controllerutils/predicate
-        - pkg/controllerutils/reconciler
-        - pkg/controllerutils/routes
-        - pkg/features
-        - pkg/gardenlet/apis/config
-        - pkg/gardenlet/apis/config/v1alpha1
-        - pkg/healthz
-        - pkg/logger
-        - pkg/resourcemanager/apis/config
-        - pkg/resourcemanager/apis/config/v1alpha1
-        - pkg/resourcemanager/apis/config/validation
-        - pkg/resourcemanager/client
-        - pkg/resourcemanager/controller
-        - pkg/resourcemanager/controller/csrapprover
-        - pkg/resourcemanager/controller/garbagecollector
-        - pkg/resourcemanager/controller/garbagecollector/references
-        - pkg/resourcemanager/controller/health
-        - pkg/resourcemanager/controller/health/health
-        - pkg/resourcemanager/controller/health/progressing
-        - pkg/resourcemanager/controller/health/utils
-        - pkg/resourcemanager/controller/managedresource
-        - pkg/resourcemanager/controller/node
-        - pkg/resourcemanager/controller/node/helper
-        - pkg/resourcemanager/controller/secret
-        - pkg/resourcemanager/controller/tokeninvalidator
-        - pkg/resourcemanager/controller/tokenrequestor
-        - pkg/resourcemanager/predicate
-        - pkg/resourcemanager/webhook
-        - pkg/resourcemanager/webhook/crddeletionprotection
-        - pkg/resourcemanager/webhook/endpointslicehints
-        - pkg/resourcemanager/webhook/extensionvalidation
-        - pkg/resourcemanager/webhook/highavailabilityconfig
-        - pkg/resourcemanager/controller/networkpolicy
-        - pkg/resourcemanager/webhook/podschedulername
-        - pkg/resourcemanager/webhook/podtopologyspreadconstraints
-        - pkg/resourcemanager/webhook/projectedtokenmount
-        - pkg/resourcemanager/webhook/seccompprofile
-        - pkg/resourcemanager/webhook/systemcomponentsconfig
-        - pkg/resourcemanager/webhook/tokeninvalidator
-        - pkg/utils
-        - pkg/utils/context
-        - pkg/utils/errors
-        - pkg/utils/flow
-        - pkg/utils/gardener
-        - pkg/utils/kubernetes
-        - pkg/utils/kubernetes/health
-        - pkg/utils/retry
-        - pkg/utils/secrets
-        - pkg/utils/timewindow
-        - pkg/utils/validation/admissionplugins
-        - pkg/utils/validation/cidr
-        - pkg/utils/validation/features
-        - pkg/utils/version
-        - vendor
-        - VERSION
+    - image: eu.gcr.io/gardener-project/gardener/operator
```

Comment:
Would it be possible to preserve the order of the keys and the indentation to shrink the diff?